s3watcher/lambda/lib/Transfer.js

```diff
@@ -41,27 +41,27 @@ module.exports = {
 
         // TODO: Use stream API
         const operation = function() {
-            Logger.logDownload(config.stage, s3Object);
-
-            return S3Helper.getS3Object(s3Object).flatMap(function(data){
-                Logger.logUpload(config.stage, {
-                    size: upload.size,
-                    filename: upload.params.filename,
-                    uploadedBy: upload.params.uploadedBy,
-                    stage: upload.params.stage
-                });
+            Logger.logImport(config.stage, s3Object);
 
-                return Upload.postData(upload, data.Body);
-            }).retry(5).flatMap(function(uploadResult){
-                Logger.logRecordToCloudWatch(config.stage, uploadResult);
+            const urlExpiryInSeconds = config.s3UrlExpiry;
```

Comment:
Standardising would be nice, but I'm not sure about the build-time option. Shall we chat around coffee?